specification/ai/Azure.AI.Projects/common/models.tsp

```diff
@@ -115,3 +115,23 @@ model AssetCredentialResponse {
   @doc("Credential info to access the storage account.")
   blobReference: BlobReference;
 }
+
+@doc("System metadata for an asset")
+model SystemData {
```

Comment:
Making use of common SystemData and CatalogInformation to be added in https://github.com/Azure/azure-rest-api-specs/pull/35559/files